maps/Arfs/residential/_residential.dm

```diff
@@ -0,0 +1,210 @@
+// -- Shuttle -- //
```

Comment:
This entire file can be deleted, as it's not in use anywhere and is just a copy of _residential_arfs.dm